presto-spi/src/main/java/com/facebook/presto/spi/type/SqlTimeWithTimeZone.java

```diff
@@ -16,30 +16,64 @@
 import com.fasterxml.jackson.annotation.JsonValue;
 
 import java.time.Instant;
+import java.time.LocalTime;
```

Comment:
cmt msg cont: 

* remove "ID" from the msg, i.e.: "Fix TZ issue in TIME WITH TZ for TZ <strike>ID</strike> changing offset over time"